packages/Card/src/Card.styles.js

```diff
@@ -0,0 +1,59 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+// Common
```

Comment:
not sure this or https://github.com/acl-services/paprika/pull/531/files#diff-b3b863891964f1b45c757d2de0e060e0R18 is necessary

Reply:
do you mean the comment?